.github/workflows/check-react-native-nightly.yml

```diff
@@ -0,0 +1,64 @@
+name: Check React Native nightly build
+on:
+  pull_request:
+    paths:
+      - .github/workflows/check-react-native-nightly.yml
+  schedule:
+    - cron: '37 19 * * *'
+  workflow_dispatch:
+
+jobs:
+  build_ios:
+    runs-on: macos-12
+    strategy:
+      matrix:
+        react-native-architecture: ['Paper', 'Fabric']
+      fail-fast: false
+    concurrency:
+      group: ios-react-native-nightly-${{ matrix.react-native-architecture }}-${{ github.ref }}
+      cancel-in-progress: true
+    steps:
+      - name: Create app
+        run: npx react-native init app --skip-install --version nightly
+      - name: Install Reanimated
+        working-directory: app
+        run: yarn add github:software-mansion/react-native-reanimated
+      - name: Install Paper Pods
+        if: ${{ matrix.react-native-architecture == 'Paper' }}
+        working-directory: app/ios
+        run: pod install
+      - name: Install Fabric Pods
+        if: ${{ matrix.react-native-architecture == 'Fabric' }}
```

Comment:
Why do we install both Paper and Fabric pods and then launch an app? Shouldn't we do it separately for each architecture?